### ISSUE TITLE
StealJS.com audit

### ISSUE THREAD

**Issue by chasenlehara** [stealjs/steal#923]
Go through the website and look for broken demos & links, missing images & videos, spelling & grammar issues, accessibility issues, poor performance, etc.

**Comment by matthewp** [comment]
Issue moved to [stealjs/stealjs #15](https://github.com/stealjs/stealjs/issues/15) via [**ZenHub**](https://www.zenhub.com/)

**Comment by matthewp** [comment]
Moved this to the stealjs repo since that is where the website is.

**Comment by chasenlehara** [comment]
Thanks, should’ve realized that!
